packages/debug/src/browser/style/stop.svg

```diff
@@ -1,4 +1,6 @@
 <!--Copyright (c) Microsoft Corporation. All rights reserved.-->
 <!--Copyright (C) 2018 TypeFox and others.-->
 <!--Licensed under the MIT License. See License.txt in the project root for license information.-->
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><defs><style>.icon-canvas-transparent,.icon-vs-out{fill:#f6f6f6;}.icon-canvas-transparent{opacity:0;}.icon-vs-action-red{fill:#a1260d;}</style></defs><title>stop</title><g id="canvas"><path class="icon-canvas-transparent" d="M16,0V16H0V0Z"/></g><g id="outline" style="display: none;"><path class="icon-vs-out" d="M14,2V14H2V2Z"/></g><g id="iconBg"><path class="icon-vs-action-red" d="M13,3V13H3V3Z"/></g></svg>
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Was this icon updated to be in-sync with the upstream repo? Or just tweaked locally? Either way I think the TypeFox copyright should list 2020.

Reply:
@JesterOrNot did you modify the icon after it was copied?
 I assume it is actually the `debug-stop.svg`, but the path is different:

https://github.com/microsoft/vscode-icons/blob/b73945c70f1117c4e65939dd3e10bdd623cb4ef3/icons/light/debug-stop.svg#L1-L3

Reply:
Done!